satellite/console/service.go

```diff
@@ -864,6 +867,23 @@ func (s *Service) CreateProject(ctx context.Context, projectInfo ProjectInfo) (p
 		return nil, ErrProjLimit.Wrap(err)
 	}
 
+	cards, err := s.accounts.CreditCards().List(ctx, auth.User.ID)
+	if err != nil {
+		s.log.Debug(fmt.Sprintf("could not add promotional coupon for user %s", auth.User.ID.String()), zap.Error(Error.Wrap(err)))
+		return nil, Error.Wrap(err)
+	}
+
+	balance, err := s.accounts.Balance(ctx, auth.User.ID)
+	if err != nil {
+		s.log.Debug(fmt.Sprintf("could not add promotional coupon for user %s", auth.User.ID.String()), zap.Error(Error.Wrap(err)))
+		return nil, Error.Wrap(err)
+	}
+
+	if len(cards) == 0 && balance < s.minCoinPayment {
+		s.log.Debug(fmt.Sprintf("could not add promotional coupon for user %s - no valid payment methods found", auth.User.ID.String()), zap.Error(Error.Wrap(err)))
+		return nil, Error.Wrap(err)
```

Comment:
you don't have error here, need to create new one

Reply:
Thanks!

Reply:
Done